### PR TITLE
update of github paths

### PR DESCRIPTION
**Pull request by ptroc**
@rebuy-de/prp-cloud-infrastructure

### CODE REVIEW

File: config/services.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 settings:
-  kubeconfig: ~/config/kubeconfig.yaml
+  kubeconfig: /home/ptroc/.kube/config
```
Comment (alexsomesan) [inline config/services.yaml]: We don't all have this :)

Comment (bjoernhaeuser) [inline config/services.yaml]: 🎱

Comment (das-vinculum) [inline config/services.yaml]: Without the specific location, should be fine.

Comment (ptroc) [inline config/services.yaml]: what about ~/.kube/kubeconfig.yml

Comment (alexsomesan) [inline config/services.yaml]: should work for all

Comment (das-vinculum) [inline config/services.yaml]: I like '~/config/kubeconfig.yaml' more, that starts to get a default configuration location, currently.

Comment (das-vinculum) [inline config/services.yaml]: After short discussion I am fine with @ptroc suggestion, if do not do it for production later on.

**Pull request merged**